vector/src/main/java/im/vector/activity/RiotAppCompatActivity.kt

```diff
@@ -293,4 +295,8 @@ abstract class RiotAppCompatActivity : AppCompatActivity() {
         ConsentNotGivenHelper(this, savedInstanceState)
                 .apply { addToRestorables(this) }
     }
+
+    protected fun applyScreenshotSecurity() {
+        window.addFlags(WindowManager.LayoutParams.FLAG_SECURE)
```

Comment:
maybe we should do this only on Release version, because when debugging, it's sometimes quite useful to take screenshot

Reply:
@af-anssi: we would prefer to apply this limitation only on Release version. Do you agree with this?